### PR TITLE
@mzikherman => [Artist Bugfixes] Misc Jira tickets

### PR DESCRIPTION
**Pull request by damassi**
#### Responsive artist page - Overview >> Exhibitions section is empty (but there are exhibitions in CV)
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-173

See https://www.artsy.net/artist/wei-tan

#### Artist >> Auction tab - Auction result CTA for logged out users needs different button styling
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-131

screen shot 2018-08-09 at 5 27 32 pm

#### Responsive artist page - Blank sub-sections within tabs should be hidden entirely
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-177

- need to qa

#### Responsive artist page - Empty tabs should be hidden
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=DISCO&modal=detail&selectedIssue=DISCO-176

(dup)

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a58b43c6c6e977bf2e6b1b08740740c90b54634d

https://deploy-preview-1110--artsy-reaction.netlify.com

File: src/Apps/Artist/Routes/Overview/index.tsx [hunk 1]
```diff
@@ -122,24 +178,49 @@ export const OverviewRouteFragmentContainer = createFragmentContainer(
...
       }
 
-      href
-      is_consignable
+      currentEvent {
+        name
+      }
 
-      ...Genes_artist
+      related {
```
Comment (mzikherman) [inline src/Apps/Artist/Routes/Overview/index.tsx]: i see, so you basically duplicate the fragments which return the actual content to each component (or in a slimmed down way, just plucking the minimal `id`), as an exist-y check.

I think this is 👍 given the current patterns and metholodgy, but it'd be great for it to be less...crufty! One possibility is to move these calls into Metaphysics and enhance `statuses` or something, which might be nice to share with other clients possibly. But that can also always be done in the future if needed.

File: src/Styleguide/Components/SelectedExhibitions.tsx [hunk 2]
```diff
@@ -119,7 +126,7 @@ export class SelectedExhibitionsContainer extends React.Component<
   }
   render() {
     if (
-      !this.props.exhibitions ||
+      !this.props.exhibitions.length ||
```
Comment (mzikherman) [inline src/Styleguide/Components/SelectedExhibitions.tsx]: 👍

**Comment by mzikherman** [comment]
Merge at will!

File: src/Apps/Artist/Routes/Overview/index.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Artist/Routes/Overview/index.tsx]: Yah, moving to MP actually makes way more sense -- this duplication was bothering me and I tried all different ways to get it to return properly. Will give it 👀

Comment (damassi) [reply src/Apps/Artist/Routes/Overview/index.tsx]: @mzikherman - I'm gonna merge to get these issues out and add a FIXME and lets pair tomorrow on this from a MP perspective, I'm curious about some approaches.

**Pull request merged**